02_end_to_end_machine_learning_project-checkpoint.ipynb

```diff
@@ -0,0 +1,5144 @@
+{
```

Comment:
<p>test comment on older commit in PR</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/handson-ml/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>